ykcompile/src/lib.rs

```diff
@@ -503,6 +507,109 @@ impl<TT> TraceCompiler<TT> {
         Ok(())
     }
 
+    fn c_checked_binop(
+        &mut self,
+        dest: &Place,
+        binop: &BinOp,
+        op1: &Operand,
+        op2: &Operand,
+    ) -> Result<(), CompileError> {
+        // Move `op1` into `dest`.
+        match op1 {
+            Operand::Place(p) => self.mov_place_place(dest, &p)?,
+            Operand::Constant(Constant::Int(ci)) => self.mov_place_constint(dest, &ci)?,
+            Operand::Constant(Constant::Bool(b)) => self.mov_place_bool(dest, *b)?,
```

Comment:
There are no `CheckedBinops` that take a bool as operand. Only ints.

Reply:
Fair enough! I can change this to `unreachable!()`.

Reply:
Fixed in b264e6c